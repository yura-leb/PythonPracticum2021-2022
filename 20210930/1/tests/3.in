99,88,77,33,11,22,66,44,55
