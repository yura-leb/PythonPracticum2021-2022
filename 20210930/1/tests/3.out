[99, 11, 55, 77, 44, 88, 66, 22, 33]
